### DISCUSSION TITLE
Participation in the project

### DISCUSSION THREAD

**Discussion by prankware-code** [dbeaver/dbeaver#23420] [DBeaver Development]
Hi, I'm a backend C++ developer with 2 years of experience. Can I help with this project?

**Comment by LonwoLonwo** [comment]
Hello @melisha-git 

Thanks for your interest in our product.

You can help us with fixing our bug tickets: https://github.com/dbeaver/dbeaver/issues?q=is%3Aissue+is%3Aopen+label%3Abug.
Or help with the product localization: https://github.com/dbeaver/dbeaver/wiki/Localization
You can take a look at tickets with the "good first issue" label:[ good first issue](https://github.com/dbeaver/dbeaver/labels/good%20first%20issue)
